TheMealDB/TheMealDB/Moduls/CategroryScreen/Controller/CategoryController.swift

```diff
@@ -40,22 +48,89 @@ class CategoryController: UIViewController {
                     return
                 }
                 self.categories = category
+                self.filterCategory = category
+            case .failure(let error): print(error as Any)
```

Comment:
```suggestion
             case .failure(let error):
                      //show alert thông báo error
```
hiển thị lỗi ở đây lên